profile_specs/profiles.json

```diff
@@ -234,7 +366,9 @@
             "name": "Latest",
             "resourceTypes": {
                 "Microsoft.AnalysisServices": {
-                    "2017-07-14": ["*"]
+                    "management": {
```

Comment:
why do we need a "management" tag here, if we already in the "management" section https://github.com/Azure/azure-sdk-for-ruby/pull/1158/commits/a818cec53fac531fce38474119376a72da2ff63b#diff-a886d4e0a33d2455b5a5344309de4389R363 ?